ui/components/ui/button/buttons.scss

```diff
@@ -302,7 +302,7 @@ input[type="submit"][disabled] {
 
 .btn--inline {
   display: inline;
-  padding: 0 4px;
+  padding: 0;
```

Comment:
@georgewrmarshall What do you think of this? I'm not sure why we would want r-l padding on inline buttons? It makes blending them with other texts more awkward.

Reply:
Hmm initially I thought "yah you're right!" but then I checked storybook and the inline button type. It seems the padding is required for context in which it is used in this story. Removing the padding removes the space altogether. Are you seeing it differently in your context?

[Inline type button in storybook](https://metamask.github.io/metamask-storybook/index.html?path=/story/ui-components-ui-button-button-stories-js--type-inline)

Removing the padding

<img width="1440" alt="Screen Shot 2022-03-23 at 2 59 13 PM" src="https://user-images.githubusercontent.com/8112138/159803365-93b3da97-01ad-406b-8fe8-ee1fdf605819.png">

Reply:
Ok to remove the padding of the inline type button after our discussions and some exploration 👍  🙏